src/client/src/containers/AzureLoginModal/strings.ts

```diff
@@ -0,0 +1,40 @@
+import { defineMessages } from "react-intl";
+
+export const strings = defineMessages({
+    freeAccountAnswer: {
+        defaultMessage: "All you need is a phone number, a credit or debit card, and a GitHub account or Microsoft account username (formerly Windows Live ID).",
+        id: "azureLoginModal.freeAccountAnswer"
+    },
+    freeAccountQuestion: {
+        defaultMessage: "What do I need to sign up for a free account?",
+        id: "azureLoginModal.freeAccountQuestion"
+    },
+    freeTrialUpgradeAnswer: {
+        defaultMessage: "TODO",
+        id: "azureLoginModal.freeTrialAnswer"
+    },
+    freeTrialUpgradeQuestion: {
+        defaultMessage: "What happens if I want to upgrade my free trial?",
+        id: "azureLoginModal.freeTrialQuestion"
+    },
+    freeTrialAnswer: {
+        defaultMessage: "We’ll notify you so you can decide if you want to upgrade to pay-as-you-go pricing and remove the spending limit. If you do, you’ll have access to all the free products. If you don’t, your account and products will be disabled, and you'll need to upgrade to resume usage.",
+        id: "azureLoginModal.freeTrialUpgradeAnswer"
+    },
+    freeTrialQuestion: {
+        defaultMessage: "Do I get charged once I finish my free trial?",
+        id: "azureLoginModal.freeTrialUpgradeQuestion"
```

Comment:
should be azureLoginModal.freeTrialQuestion

Reply:
PR updated :)